scripts/exgdas_enkf_update.sh

```diff
@@ -179,9 +183,14 @@ else
 fi
 nfhrs=$(echo $IAUFHRS_ENKF | sed 's/,/ /g')
 for imem in $(seq 1 $NMEM_ENS); do
+   smem=$((imem + mem_offset))
+   if (( smem > 80 )); then
+      smem=$((smem - 80))
+   fi
```

Comment:
```suggestion
   smem=$(( (imem + mem_offset) % 80 ))
```

Reply:
This change causes the scripts to fail. When `imem + mem_offset = 80` this causes the result of `smem=0` and causes the script to replace member 80 with member 00. Member 80 should still exist and then when `imem + mem_offset = 81` gets calculated it should go to member 1 instead of 0.

Reply:
Yeah, you're right. Stupid 1-indexed arrays. Any way that keeps the modulus and works around it is just messier, so let's keep it how it was.